unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneBoundariesController/SceneBoundsFeedbackStyle_Simple.cs

```diff
@@ -6,6 +6,7 @@ namespace DCL.Controllers
 {
     public class SceneBoundsFeedbackStyle_Simple : ISceneBoundsFeedbackStyle
     {
+        //TODO: This is not working correctly because it only disable the first renderer instead of all renderers
```

Comment:
```suggestion
```
We already analyzed this and this is not the case (The callback is called for all renderers from outside). We should remove this comment.

Reply:
It is removed in the refactor PR